sample/src/main/java/com/stepstone/stepper/sample/nofrag/NoFragAdapter.java

```diff
@@ -0,0 +1,63 @@
+package com.stepstone.stepper.sample.nofrag;
+
+import android.content.Context;
+import android.support.v4.view.ViewPager;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.stepstone.stepper.Step;
+import com.stepstone.stepper.adapter.AbstractStepAdapter;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Created by leonardo on 18/12/16.
+ */
+
+public class NoFragAdapter extends AbstractStepAdapter<NoFragView> {
```

Comment:
In general it might be good to make AbstractStepAdapter be declared as:

`public abstract class AbstractStepAdapter<T extends View & Step> extends PagerAdapter implements StepAdapter<T>`

similar to how the Fragment version is declared?